src/components/MeasurementsToolList.vue

```diff
@@ -39,59 +39,80 @@ const remove = (id: ToolID) => {
 const jumpTo = (id: ToolID) => {
   props.toolStore.jumpToTool(id);
 };
+
+const toggleHidden = (id: ToolID) => {
+  const toggled = !props.toolStore.toolByID[id].hidden;
+  props.toolStore.updateTool(id, { hidden: toggled });
+};
 </script>
 
 <template>
-  <v-list-item v-for="tool in tools" :key="tool.id" lines="two">
-    <template #prepend>
-      <v-icon class="tool-icon">{{ icon }}</v-icon>
-      <div class="color-dot mr-3" :style="{ backgroundColor: tool.color }" />
-    </template>
-    <v-list-item-title v-bind="$attrs">
-      {{ tool.labelName }}
-    </v-list-item-title>
+  <v-list-item v-for="tool in tools" :key="tool.id">
+    <v-container>
+      <v-row class="align-center main-row">
+        <v-icon class="tool-icon">{{ icon }}</v-icon>
+        <div class="color-dot mr-3" :style="{ backgroundColor: tool.color }" />
+
+        <v-list-item-title v-bind="$attrs">
+          {{ tool.labelName }}
+        </v-list-item-title>
 
-    <v-list-item-subtitle>
-      <slot name="details" v-bind="{ tool }">
-        <v-row>
-          <v-col>Slice: {{ tool.slice + 1 }}</v-col>
-          <v-col>Axis: {{ tool.axis }}</v-col>
-        </v-row>
-      </slot>
-    </v-list-item-subtitle>
-    <template #append>
-      <v-btn
-        class="mr-2"
-        icon="mdi-target"
-        variant="text"
-        @click="jumpTo(tool.id)"
-      >
-        <v-icon>mdi-target</v-icon>
-        <v-tooltip location="top" activator="parent"> Reveal Slice </v-tooltip>
-      </v-btn>
-      <v-btn icon="mdi-delete" variant="text" @click="remove(tool.id)">
-        <v-icon>mdi-delete</v-icon>
-        <v-tooltip location="top" activator="parent">Delete</v-tooltip>
-      </v-btn>
-    </template>
+        <span class="ml-auto actions">
+          <v-btn icon variant="text" @click="toggleHidden(tool.id)">
+            <v-icon v-if="tool.hidden">mdi-eye-off</v-icon>
+            <v-icon v-else>mdi-eye</v-icon>
+            <v-tooltip location="top" activator="parent">{{
+              tool.hidden ? 'Show' : 'Hide'
+            }}</v-tooltip>
+          </v-btn>
+          <v-btn icon variant="text" @click="jumpTo(tool.id)">
+            <v-icon>mdi-target</v-icon>
+            <v-tooltip location="top" activator="parent"
+              >Reveal Slice</v-tooltip
```

Comment:
Got some formatting snafu here. I'm still not sure why prettier does this.

Reply:
👍️ 
🧓 Lines don't grow on trees sonny.